targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_KL43Z/TARGET_FRDM/PinNames.h

```diff
@@ -134,6 +134,11 @@ typedef enum {
     A4 = PTC2,
     A5 = PTC1,
 
+    SPI_MOSI    = PTD6,
+    SPI_MISO    = PTD7,
+    SPI_SCK     = PTD5,
+    SPI_PERSISTENT_MEM_CS = PTD4,
```

Comment:
@dannybenor @ithinuel we should be documenting all the names for pins and "components" in our porting guide so people have standard they can follow while porting platform. @ithinuel is working on defining new SPI spec, can we make sure that all the required names are mentioned there.

Reply:
Documentation of handbook and porting guide will be done.

Reply:
IHMO defining such name is error prone for this kind of bus/component.
How would you handle targets with multiple Flash on multiple SPI interfaces?

Typically, this PR makes the config for "COMPONENT_SD" on the NUCLEO_F429ZI use:
```
"SPI_MOSI": "PC_12",
"SPI_MISO": "PC_11",
"SPI_CLK":  "PC_10",
"SPI_CS":   "PA_15"
```
while on my board+shield (https://github.com/armmbed/ci-test-shield) I actually need :
```
"SPI_MOSI": "PB_5",
"SPI_MISO": "PA_6",
"SPI_CLK":  "PA_5",
"SPI_CS":   "PD_14"
```
(see also https://os.mbed.com/teams/ST/wiki/Nucleo-144pins-ethernet-spi-conflict)

Reply:
@ithinuel The goal of the feature is to define the **default** for "SPI_MOSI" like pins. You still can override the default with a different SPI bus or CS.

Reply:
@dannybenor What I meant is that providing such **default** value inherently gives them more *truthiness* and I wonder on what is this based ?
eg:
> This is the onboard Flash chip.

or
> This is the vendor's demo setup.